### PR TITLE
Chinese Version on Introduction

### PR DESCRIPTION
**Pull request by adadai88**
Please check if the added document is a correct PR for further review. thanks!  @rrrutledge

### CODE REVIEW

**Comment by rrrutledge** [comment]
Thanks so much, @adadai88!  @tanzhongyibidu (or other native speakers), are you able to have a look?

File: introduction/zh/02-InnerSource解决的问题.asciidoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+=== InnerSource解决的问题?
+
+无论员工在公司组织结构中的位置如何，InnerSource鼓励并奖励与任何人的协作和代码的重复利用。这种方法与传统组织中看到的不同，传统组织中的思想和工作产品往往被困在内部企业层级结构及约束的边界内。让我们来探讨一个例子来说明这个想法。
+
+想象一下，同一公司的两个团队交付不同的软件，其中一个团队的软件依赖于另一个团队的软件。例如，用户体验依赖于API服务来检索显示的数据。这种情况在大型企业中很常见，在大型企业中，单个团队生产的软件可能有几十个或几百个消费者。
+
+当消费团队需要许多特性时，生产团队通常有一些需求和优先级流程来决定他们将使用哪些特性。对于重要的特性请求，如果没有按照当前工作的优先级排序，消费团队通常会选择3个选项中的一个，每个选项都有自己的缺点。
```
Comment (adadai88) [inline introduction/zh/02-InnerSource解决的问题.asciidoc:7]: 消费团队过于直译， 考虑使用”使用团队”作为替代
it is better we use "使用团队"instead of "消费团队"

Comment (tanzhongyibidu) [reply introduction/zh/02-InnerSource解决的问题.asciidoc:7]: 使用团队要好些， host team是不是也翻译为维护团队？

**Comment by adadai88** [comment]
i have reviewed and revised this section again, LGTM

File: introduction/zh/02-InnerSource解决的问题.asciidoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+=== InnerSource解决的问题?
+
+无论员工在公司组织结构中的位置如何，InnerSource鼓励并奖励与任何人的协作和代码的重复利用。这种方法与传统组织中看到的不同，传统组织中的思想和工作产品往往被困在内部企业层级结构及约束的边界内。让我们来探讨一个例子来说明这个想法。
+
```
Comment (tanzhongyibidu) [inline introduction/zh/02-InnerSource解决的问题.asciidoc:4]: 代码的重复利用， 可以改为代码的重用。
一来“重用”用的多些，重复利用少些，二来和之前的“任何人的协作”对应上

**Comment by lenucksi** [comment]
Thanks for doing the review work - could you please click "Resolve conversation" (or the translated equivalent) on all open discussions/review parts once they are done? 
I would like to not merge while discussions are "open" since I can not judge if I would overlook anything as I can not read the contents.

Please also indicate explicitly when I should merge.

(Other than that: Thanks again @adadai88 and @tanzhongyibidu!)

**Comment by rrrutledge** [comment]
Agree!  Glad you can all work together and please let us know using the _GitHub_ indicators that @lenucksi mentioned when thing are good to merge.

File: introduction/zh/02-InnerSource解决的问题.asciidoc (hunk 2 above)
Comment (adadai88) [reply introduction/zh/02-InnerSource解决的问题.asciidoc:4]: noted, done

**Comment by adadai88** [comment]
Jerry Tan  4:29 PM
The purpose of this document is the hold the supplementary and planning material related to the content of the introductory segments of this learning path.
本文件旨在保存与本学习路径导论部分内容相关的补充和计划材料。

4:31
Introduce the topic, why it’s important, and what the viewer will learn in this learning path.
介绍主题即InnerSource的重要性，以及观众在学习过程中将学到什么。
4:33
Share a hypothetical (yet believable) example situation that illustrates the difficulties today in the off-the-shelf process of collaborating across organizational silos. 
分享一个假设的（但可信的）例子，它说明了当前跨越组织壁垒而进行协作的当前过程中的困难。

**Comment by adadai88** [comment]
> 旨在

fixed

File: introduction/zh/提纲.asciidoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
+:doctype: book
```
Comment (rrrutledge) [inline introduction/zh/提纲.asciidoc:1]: What English file was the base for this file?

Comment (adadai88) [reply introduction/zh/提纲.asciidoc:1]: https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/introduction/outline.asciidoc   check this Russ

**Comment by lenucksi** [comment]
> LGTM。
> I have discussed with ada about the patch offline last week。

Excellent, thanks for giving a green light @tanzhongyibidu and thanks again to both of you for creating this translation! :tada: 
I'll then press the big green button and merge it!

File: introduction/zh/提纲.asciidoc (hunk 3 above)
Comment (rrrutledge) [reply introduction/zh/提纲.asciidoc:1]: Got it - thank you!

**Comment by rrrutledge** [comment]
So awesome!

**Comment by tanzhongyibidu** [comment]
thanks everyone.

**Comment by rrrutledge** [comment]
🙌

**Pull request merged**